### ISSUE TITLE
Update webservice

### ISSUE THREAD

**Issue by DeniseSl22** [wikipathways/wikipathways.org#84]
Hi all,

Since Jonathan asked for the last webservice update in terms of BridgeDb mappings, I've created several new metabolite mappings files. Would be great if the latest one could be behind the content of WPs (see https://www.bridgedb.org/mapping-databases/hmdb-metabolite-mappings/ ). However, we've been experiencing some problems creating a new mapping file for genes/proteins from Biomart, so that will either have to be done separately, or do the update in one go. I create a new metabolite mapping files ~ once a month, the gene protein should be once every 3 months... I'm not sure who took over the webservice updating stuff from Jonathan, but I heard that the server is in SF, so I need someone from the team there to help me :) @AlexanderPico @khanspers @ariutta @egonw @mkutmon

**Comment by egonw** [comment]
I like to note here too that we cannot reproduce right now the building of new gene/protein identifier databases, see https://github.com/bridgedb/create-bridgedb-genedb/issues/4 and https://github.com/bridgedb/create-bridgedb-genedb/issues/5

**Comment by Chris-Evelo** [comment]
I am not sure, but I think we discussed this before and decided that we would do these updates whenever we did a WikiPathways release anyway, thus making sure that mappings on the active server would not change between releases. I think that would make sense even if we did not discuss that before.

On the other hand, whenever we do a new WikiPathways release it should be fine to use the latest, tested mappings. If the metabolite mappings are newer than the gene-product mappings that should not be a problem.
